mod/wikirate/features/claim.feature

```diff
@@ -57,6 +57,7 @@ Feature: note
     And I fill in "sourcebox" with "http://google.com/?q=ymca"
     And I press "Submit"
     And I wait until ajax response
+    And I wait until ajax response
```

Comment:
One wait for adding the source and one for submitting. Could be that it was just luck that it worked.